doc/appendices/command-line/traffic_layout.en.rst

```diff
@@ -111,6 +111,12 @@ Subcommands options
 
       Fix the permission issues verify found. ``--fix`` requires root privilege (sudo).
 
+- Copy style option: ::
```

Comment:
Use Sphinx support for commands and options - see the other "traffic_..." files for examples.

Reply:
I'll take a look